src/main/java/qna/domain/Question.java

```diff
@@ -92,4 +94,33 @@ public boolean isDeleted() {
     public String toString() {
         return "Question [id=" + getId() + ", title=" + title + ", contents=" + contents + ", writer=" + writer + "]";
     }
+
+    public void checkOwner(User loginUser) throws CannotDeleteException {
+        if(!isOwner(loginUser)){
```

Comment:
조건문 사용시 다음과 같이 if 와 괄호 사이에 공백을 넣어주시는게 컨벤션이오니 참고 부탁드려요 :)
[7.4절](https://myeonguni.tistory.com/1596)
https://naver.github.io/hackday-conventions-java/#space-between-keyword-parentheses
```suggestion
        if (!isOwner(loginUser)) {
```

Reply:
네 정보 감사합니다 :)